pyproject.toml

```diff
@@ -17,8 +16,26 @@ pydot = "^1.4.2"
 graphviz = "^0.20.1"
 lxml = "^4.9.2"
 seaborn = "^0.12.2"
+
+# Issue between poetry and tensorflow metadata since >=2.11
+# This is a temporary workaround
+# related to https://github.com/python-poetry/poetry/issues/8271
+# Inspired from https://github.com/tensorflow/tensorflow/blob/adb39b04e9cb116df4659a7e2de9eea27e62f25c/tensorflow/tools/pip_package/setup.py#L148-L162
+tensorflow = {version = "2.15.0" }
+tensorflow-macos = { version = "2.15.0", platform = "darwin" }
+tensorflow-cpu = [
```

Comment:
This line will throw an error in prospector linting. Somehow it doesn't agree with it being a list.

Reply:
Strange indeed.